frontend/src/metabase/visualizations/components/ChartCaption.unit.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ComponentPropsWithoutRef } from "react";
+import { noop } from "underscore";
```

Comment:
I believe we usually just import it like `import _ from "underscore"`

Reply:
it's not the best approach to load all the methods from the lib just to use one, it can affect start time of jest and blow bundle, but I believe this problem should be addressed separately 🙇